apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -410,29 +419,33 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
     if (numberOfNewFiles === 0) {
       instructionalText = fileModeLocked ? (
         <React.Fragment>
-          There were no new {headerModeText} CVR files automatically found on
-          this USB drive. Save CVR files to this USB drive from the scanner.
-          Optionally, you may manually select files to load.
+          No new {headerModeText.toLowerCase()} CVR exports were automatically
+          found on this USB drive.
```

Comment:
Erring on the side of more concise text since I know longer text often isn't read